tsconfig.json

```diff
@@ -27,7 +27,7 @@
     // "useDefineForClassFields": true,                  /* Emit ECMAScript-standard-compliant class fields. */
 
     /* Modules */
-    "module": "ES6" /* Specify what module code is generated. */,
+    "module": "ESNext" /* Specify what module code is generated. */,
```

Comment:
Why ESNext? I'd say stay with `ES6`, if we can, since otherwise somebody might make an issue saying that Mermaid no longer works on Internet explorer.

(maybe we should consider upgrading to ES2020 or ES2022 in a future Mermaid v10 release)

Reply:
This was changed to support the json import in .vite/build.ts 

Maybe I should add a tsconfig in .vite to override just there?

Reply:
It's probably easier to just replace the JSON import, since it's not officially supported in the JavaScript spec (using `ESNext` is always risky).

```js
import { readFile } from 'node:fs/promises';
const {packages} = JSON.parse(await readFile("../package.json", {encoding: "utf8"}));
```